### DISCUSSION TITLE
How to limit range selection?

### DISCUSSION THREAD

**Discussion by heysahilsingh** [dmuy/duDatepicker#56] [Q&A]
I need to limit the range selection by 7 days. This means I can only select up to 7 days in the range. How can I?

**Comment by dmuy** [comment]
Hi @heysahilsingh, unfortunately there isn't a built-in option for this currently. But I plan to add this in the future release. Thank you.

**Reply by heysahilsingh** [reply]
oh ho, this lib seems to be perfect, it would be great if this option was added. Well, isn't there any way to achieve this? i mean I need to limit the range selection,
